travis/5.5.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+yes | pecl install -f amqp-1.6.1
```

Comment:
With php-amqp >= 1.7.0 both PHP 7 and PHP 5 supported, so feels free to install 1.7.0. It is even recommended for the sake of API consistency while 1.7.0 introduces new API methods and extend or cleanup existent one.